syntaxes/tests/vba/other.bas

```diff
@@ -6,6 +6,35 @@ Attribute VB_Name = "SyntaxTest"
   ' Some comment
 ' ^^^^^^^^^^^^^^ comment.line.quote
 
+10 ' Some comment with line number
```

Comment:
The number could be tested with the following:
```vba
10 ' Some comment with line number
' <--
```

`<` indicates we match from the line start and the number of dashes specifies the length.
You can change this if you want.

Reply:
Ok, that's good to know. I don't see a reason not to change it.